packages/playwright-core/src/server/trace/viewer/traceViewer.ts

```diff
@@ -146,6 +146,7 @@ export async function openTraceViewerApp(traceUrls: string[], browserName: strin
     ignoreDefaultArgs: ['--enable-automation'],
     colorScheme: 'no-override',
     useWebSocket: isUnderTest(),
+    keepDefaultDownloadBehavior: isUnderTest() ? false : true,
```

Comment:
Perhaps it's time to introduce `launchApp()` internal method that would also replace `installAppIcon()` and unify the usages? Because I assume all other apps we launch would also need this download behavior.

Reply:
https://github.com/microsoft/playwright/pull/26499